src/cmd/linuxkit/spec/cache.go

```diff
@@ -0,0 +1,18 @@
+package spec
+
+import (
+	"io"
+
+	"github.com/containerd/containerd/reference"
+	"github.com/google/go-containerregistry/pkg/v1"
+)
+
+type CacheProvider interface {
```

Comment:
`./spec/cache.go:10:6: exported type CacheProvider should have comment or be unexported`